hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java

```diff
@@ -2427,6 +2449,98 @@ public boolean addElasticsearch(Project project, ProjectServiceEnum serviceEnum)
       }
 
       return kibanaPatternCreated;
+    } else if(serviceEnum.equals(ProjectServiceEnum.EXPERIMENTS)) {
+
+      Map<String, String> params = new HashMap<>();
+
+      String indexName = project.getName() + "_experiments";
+
+      params.put("op", "PUT");
+      params.put("project", indexName);
+      params.put("resource", "");
+      params.put("data", "{}");
+
+      JSONObject resp = elasticController.sendElasticsearchReq(params);
```

Comment:
@robzor92 Can you try to use the elastic client for this? An example for deletion is [here](https://github.com/logicalclocks/hopsworks/blob/master/hopsworks-common/src/main/java/io/hops/hopsworks/common/elastic/ElasticController.java#L291), for creating should be similar